sdk/metricsadvisor/ai-metrics-advisor-rest/test/public/util/recordedClient.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/// <reference lib="esnext.asynciterable" />
+
+import "./env";
+
+import MetricsAdvisorRestClient, { MetricsAdvisorRestClientLike } from "../../../src";
```

Comment:
Before merging these changes we'll need to migrate to the new recorder:
https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/MIGRATION.md

Reply:
Will do it in future